src/sizzle.js

```diff
@@ -880,100 +907,106 @@ setDocument = Sizzle.setDocument = function( node ) {
 
 	// Document order sorting
 	sortOrder = hasCompare ?
-	function( a, b ) {
+		function( a, b ) {
```

Comment:
Can you de-indent that whole big block (both functions)? I know it looks weird but I'd just want to avoid massive indentation changes at this point so that `git blame` works as best as it could.